workshop/content/30-python/30-hello-cdk/200-lambda.md

```diff
@@ -198,7 +198,7 @@ Let's go to the AWS Lambda Console and test our function.
 
     ![](./lambda-2.png)
 
-4. Select __Amazon API Gateway AWS Proxy__ from the __Event template__ list.
+4. Select __API Gateway AWS Proxy__ from the __Event template__ list.
```

Comment:
This change would also be necessary in the other workshops - at the least it is in TypeScript. Could you check the others and update there as well?